lib/puppet/functions/stdlib/nested_values.rb

```diff
@@ -1,18 +1,17 @@
 # frozen_string_literal: true
 
 # This function will return list of Hash values, the return value will be Array
-# NOTE : This function is expecting only Hash and return value will be Array
 #
 # @example :
```

Comment:
I think this will render with the example having the literal name `:`

```suggestion
# @example Some text to describe the example
```

Reply:
It might also be in the wrong place.  Not sure.  Maybe needs to go directly above the dispatch (along with an `@param` tag and a `@return` tag)

Reply:
I always find I have to iterate a few times getting the puppet-strings stuff to correctly generate something nice.
Here's an [example](https://github.com/puppetlabs/puppetlabs-stdlib/blob/4120eb26fd51b6197528cf0016ecf3285a3819ba/lib/puppet/functions/stdlib/xml_encode.rb) I think has everything in the right place.